core/jobs/batch_jobs/audit_rule_validation_jobs_test.py

```diff
@@ -0,0 +1,642 @@
+              'x': 'ca_choices_68',
+              'y': 'ca_choices_68'
+            }
+          },
+          {
+            'rule_type': 'IsEqualToOrdering',
+            'inputs': {
+              'x': [
+                [
+                  'ca_choices_17', 'ca_choices_18'
+                ],
+                [
+                  'ca_choices_19'
+                ],
+                [
+                  'ca_choices_20'
+                ]
+              ]
+            }
+          },
+          {
+            'rule_type': 'IsEqualToOrderingWithOneItemAtIncorrectPosition',
+            'inputs': {
+              'x': [
+                [
+                  'ca_choices_68'
+                ],
+                [
+                  'ca_choices_69'
+                ],
+                [
+                  'ca_choices_70'
+                ]
+              ]
+            }
+          },
+          {
+            'rule_type': 'IsEqualToOrdering',
+            'inputs': {
+              'x': []
+            }
+          }
+        ],
+        'outcome': {
+          'dest': 'A valid state',
+          'feedback': {
+            'content_id': 'feedback_72',
+            'html': '<p>dvds</p>'
+          },
+          'labelled_as_correct': False,
+          'param_changes': [],
+          'refresher_exploration_id': None,
+          'missing_prerequisite_skill_id': None,
+          'dest_if_really_stuck': None
+        },
+        'training_data': [],
+        'tagged_skill_misconception_id': None
+      },
+      {
+        'rule_specs': [
+          {
+              'rule_type': 'IsEqualToOrdering',
+              'inputs': {
+                'x': [
+                  [
+                    'ca_choices_17', 'ca_choices_18'
+                  ],
+                  [
+                    'ca_choices_19'
+                  ],
+                  [
+                    'ca_choices_20'
+                  ]
+                ]
+              }
+            }
+        ],
+        'outcome': {
+          'dest': 'EXP_1_STATE_1',
+          'feedback': {
+            'content_id': 'feedback_72',
+            'html': '<p>dvds</p>'
+          },
+          'labelled_as_correct': False,
+          'param_changes': [],
+          'refresher_exploration_id': None,
+          'missing_prerequisite_skill_id': None,
+          'dest_if_really_stuck': None
+        },
+        'training_data': [],
+        'tagged_skill_misconception_id': None
+      }
+    ],
+      'default_outcome': {
+        'dest': 'EXP_1_STATE_1',
+        'feedback': {
+          'content_id': 'default_outcome',
+          'html': '<p>sd</p>'
+        },
+        'labelled_as_correct': False,
+        'param_changes': [],
+        'refresher_exploration_id': None,
+        'missing_prerequisite_skill_id': None,
+        'dest_if_really_stuck': None
+      },
+      'confirmed_unclassified_answers': [],
+      'hints': [],
+      'solution': None
+    }
+
+    # Continue Interaction.
```

Comment:
This is for the continue interaction, to get the language code for the errored exp.